static/app/views/settings/organizationIntegrations/integrationReposAddRepository.tsx

```diff
@@ -144,8 +143,8 @@ export function IntegrationReposAddRepository({
       <DropdownAutoComplete
         items={dropdownItems}
         onSelect={addRepo}
-        onChange={searchResult.searchable ? handleSearchRepositories : undefined}
-        emptyMessage={t('No repositories available')}
+        onChange={handleSearchRepositories}
+        emptyMessage={t('Please type to search for repositories')}
```

Comment:
nit: let's say something on repo name and something to say this not an end action. Like "Please enter a repository name..." or something.